bot/extensions/ping.py

```diff
@@ -0,0 +1,18 @@
+from discord.ext.commands import Cog, command
+from discord import Embed
+
+class Ping(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='ping', help='Send pong', usage='pong')
```

Comment:
**Help**

The help could be more explicite like `Shows the bot latency` or something like that.

---

**Usage**

 Also the usage isn't pong. It's ping.